user/pages/03.commerce2/03.user-guide/03.products/06.delete-a-product/docs.md

```diff
@@ -6,7 +6,13 @@ taxonomy:
 
 Sometimes you may wish to delete a product or one of its variations. Before deleting a product or variation, consider unpublishing the product or disabling the variation.
 
-! @todo the unpublish process is just a checkbox, review. No way to disable variations by default.
+## Disabling a product variation
+
+This will remove the product variation from the site, but will leave it in the system should you want to re-enable it at a later point.
+ 
+A product's variation is disabled while editing the product. Click on the variation's **Edit** button. Locate the **Active** checkbox on the Variation form and uncheck it. Click **Update Variation** to complete the process. The product variation will no longer be available to purchase on your site.

```

Comment:
> A product's variation is disabled while editing the product.

How do we describe this process in the manage a product/edit a product? We cover this partially, just not the explicit disable tidbit.

Reply:
It's not really described how to disable/remove a variant, people may look for it in either/both as it's editing and not deleting. Whatever you think.

It's all pretty obvious in the UI really but I decided to contribute this by starting at the first User doc page and going through until I came to an @TODO :-)

PR to arrive shortly.